pwchem/protocols/VirtualDrugScreening/protocol_ligand_characterization.py

```diff
@@ -0,0 +1,144 @@
+# **************************************************************************
+# *
+# * Authors: ...
+# *
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307 USA
+# *
+# * All comments concerning this program package may be sent to the
+# * e-mail address 'you@yourinstitution.email'
+# *
+# **************************************************************************
+
+import numpy as np
+import os
+from rdkit import Chem
```

Comment:
The protocol is run in the scipion3 environment, which will not have rdkit package by default. 
For using rdkit, we have a separated environment. Typically, we would call a script that contains all the rdkit logic from the protocol, in the rdkit env. You can check othe rdkit protocols to see how and the rdkit script should be placed in the pwchem/scripts directory.